apps/src/aichat/views/InfoTooltipIcon.tsx

```diff
@@ -0,0 +1,40 @@
+import classNames from 'classnames';
+import React from 'react';
+
+import FontAwesomeV6Icon from '@cdo/apps/componentLibrary/fontAwesomeV6Icon/FontAwesomeV6Icon';
+import {TooltipProps, WithTooltip} from '@cdo/apps/componentLibrary/tooltip';
+
+import styles from './info-tooltip-icon.module.scss';
+
+interface InfoTooltipIconProps {
+  id: string;
+  tooltipText: string;
+  direction?: TooltipProps['direction'];
+}
+
+const InfoTooltipIcon: React.FunctionComponent<InfoTooltipIconProps> = ({
```

Comment:
This is slick! Is there anything like this in the design system @levadadenys ? I bet this would be nice to have for other teams as well.